groups_admin.rb

```diff
@@ -100,34 +125,180 @@ def update_accept_header(extension,mime_type)
 # groups/<group id> (PUT GET DELETE)
 
 ################ groups ###############
-
+# create group representations
+# input:  https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups
```

Comment:
Not sure about the label "input" and "output" here. Maybe both links can be listed as reference?